front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -206,39 +206,18 @@ export function AgentMessage({
         });
         break;
 
-      case "agent_generation_success":
-        setStreamedAgentMessage((m) => {
-          return { ...m, content: event.text };
-        });
-        break;
-
       case "agent_chain_of_thought":
```

Comment:
we don't need to handle this event on the frontend at all. We just receive tokens as chain of thoughts as they are streamed.